contributing/content-style-guide.md

```diff
@@ -18,11 +18,60 @@ This guide includes standards we want to emphasize, likely because we've made de
+
+- Keep the file name as short as possible since it becomes part of the URL and leave out any unnecessary words. Long URLs are hard to read and prone to errors.
+- Think about where you want to locate the file in the leftbar. This will help catch and avoid redundancy. Remain consistent with the surrounding leftbar items.
+- Be as descriptive as possible so readers have an idea as to what they’re about to read.
+
+| ✅ User | ❌ Avoid|
+|:-------:|:-------:|
+|`/docs/deploy/architecture` | `/docs/deploy/about-architecture`|
+
+
+## Using Markdown
+
+docs.getdbt.com uses its own CSS, and Docusaurus supports its own specific Markdown syntax. The following table provides an overview of the supported syntax elements.
+
+| Element                                     | Syntax                                                |
+|---------------------------------------------|-------------------------------------------------------|
+| Headings                                    | `# H1`, `## H2`, `### H3`                             |
+| Bold                                        | `**bold text**`                                       |
+| Italic                                      | `_italicized text_`                                    |
+| Ordered List                                | `1.` First item (use `1.` for each item)              |
+| Unordered List                              | `-` or `*` (for each item)                            |
+| Code or command in a sentence               | ``code``                                              |
+| Link - external site                        | `[Title](https://www.example.com)`                    |
+| Link - topic in same folder                 | `[Title](/folder/file-name) without file extension`            |
+| Link - topic in different folder            | `[Title](/folder/file-name) without file extension` |
+| Link - section in topic in same folder      | `[Title](/folder/file-name#section-name)`                     |
+| Link - section in topic in different folder | `[Title](/folder/file-name#section-name)`           |
+| Image                                       | `<Lightbox src="/img/docs/<image-name>.jpg" title="Concise description of image"/>`      |
+
 ## Callouts
 
 Callouts highlight important or high-value information that readers need to know. We want callouts to stand out, so we should keep their content to a minimum, avoiding general information, permissions, or prerequisites. Too much information can make it difficult to absorb. Imagine driving down one block with five stop signs!?!
 
-Use callouts sparingly for high-value information &mdash; avoid including general information, permissions, or prerequisites in callouts.
+Use callouts sparingly for high-value information &mdash; avoid including general information, permissions, or prerequisites in callouts. Below are the some of the typical callout formats we use:
```

Comment:
This looks great! We might also want to say that if you add text after the first code, like this: `:::note This is a note title` it shows up as a title for the note.